apps/api/src/app/shared/services/logger.service.ts

```diff
@@ -1,14 +1,20 @@
 import { createNestLoggingModuleOptions, PinoLogger } from '@novu/application-generic';
 import packageJson from '../../../../package.json';
 
-export const getLogger = (context: string) => {
-  const logger = new PinoLogger(
-    createNestLoggingModuleOptions({
-      serviceName: packageJson.name,
-      version: packageJson.version,
-    })
-  );
-  logger.setContext(context);
-
-  return logger;
+let loggerInstance: PinoLogger | null = null;
+
+export const getLogger = () => {
+  if (!loggerInstance) {
+    loggerInstance = new PinoLogger(
+      createNestLoggingModuleOptions({
+        serviceName: packageJson.name,
+        version: packageJson.version,
+        silent: true,
+      })
+    );
+  }
+
+  loggerInstance.setContext('Application');
```

Comment:
due to the singleton nature of the logger instance, we use a single context to reduce the possibility of it being overridden by different clients.

this approach is fine because the logger is only used in two areas: application bootstrap and e2e.
by using a defined context, we can still monitor those logs effectively.